drivers/bridge/interface.go

```diff
@@ -61,3 +62,17 @@ func (i *bridgeInterface) addresses() (netlink.Addr, []netlink.Addr, error) {
 	}
 	return v4addr[0], v6addr, nil
 }
+
+func (i *bridgeInterface) programIPv6Address() error {
+	_, nlAddressList, err := i.addresses()
+	if err != nil {
+		return &IPv6AddrAddError{IP: i.bridgeIPv6, Err: fmt.Errorf("failed to retrieve address list: %v", err)}
+	}
+	nlAddr := netlink.Addr{IPNet: i.bridgeIPv6}
+	if !findIPv6Address(nlAddr, nlAddressList) {
```

Comment:
Can we rework this to look like this?
```
if findIPv6Address() {
    return nil
}

if err := netlink.AddrAdd(); err != nil {
   return &ipv6addradderror
}

return nil
```

This is to conform to go style of coding to keep the non-error logic in the first indent. It reads better than having to find the netlink.AddrAdd squirreled away someewhere.